docs/README.adoc

```diff
@@ -343,6 +343,20 @@ openstack overcloud deploy \
 [NOTE:] 
 The SSL certificates for the `MetricsQdr` service is configured to generate only for the `InternalApi` network but the default Ceph role does not use the `InternalApi` network. To deploy SAF client when InternalTLS is enabled, use this workaround: pass the custom Ceph role, which has `InternalApi` network, to `openstack overcloud deploy` when InternalTLS is enabled in the deployment.
 
+[id=`configuration-for-distributed-compute-nodes_{context}`]
+=== Configuration for distributed compute nodes
+If you are deploying with compute nodes on different networks to the controller, some additional changes are need so that the metrics sent over AMQ can reach the SAF.
```

Comment:
If you are deploying compute nodes on networks different from the controllers, some additional changes are required to allow data transported via AMQ Interconnect to reach the SAF server instance.